REVIEWERS.md

```diff
@@ -7,7 +7,8 @@ Please keep the below list sorted in ascending order.
 ## Reviewers
 
 | Approver        | GitHub ID           | Affiliation             | Owners files                                                                                                                                   |
-| --------------- | ------------------- |-------------------------| ---------------------------------------------------------------------------------------------------------------------------------------------- |
+|-----------------|---------------------|-------------------------|------------------------------------------------------------------------------------------------------------------------------------------------|
+| Rentian Zhou    | @Vacant2333         |                         | karmada-io/karmada test/                                                                                                                       |
```

Comment:
The list of reviewers is arranged in alphabetical order. You can put your name in the appropriate sequence.